oaipmhTemplates/dha_RepoObject.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE RDF [
+    <!ENTITY xml "http://www.w3.org/XML/1998/namespace">
+    <!ENTITY xsd "http://www.w3.org/2001/XMLSchema#">
+    <!ENTITY owl "http://www.w3.org/2002/07/owl#">
+    <!ENTITY skos "http://www.w3.org/2004/02/skos/core#">
+    <!ENTITY dc "http://purl.org/dc/elements/1.1/">
+    <!ENTITY dct "http://purl.org/dc/terms/">
+    <!ENTITY foaf "http://xmlns.com/foaf/0.1/">
+    <!ENTITY premis "http://www.loc.gov/premis/rdf/v1#">
+    <!ENTITY dha "https://digital-humanities.at/schema#">
+    <!ENTITY ebucore "http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#">
+]>
+<rdf:RDF xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:xml="&xml;" xmlns:xsd="&xsd;"
+    xmlns:skos="&skos;" xmlns:dc="&dc;" xmlns:dct="&dct;" xmlns:foaf="&foaf;" xmlns:dha="&dha;" xmlns:premis="&premis;">
+
+    <!--
+    ////////////////////////////////////////////////////////////////////////////
+    //
+    // Mapping for RepoObject
+    //
+    ////////////////////////////////////////////////////////////////////////////
+    -->
+
+    <dha:RepoObject> <!-- Q: how do I include rdf:about with URI of the Resource? -->
```

Comment:
With `val="URI"` but the problem is probably you would like to have it as an XML Node attribute value and not as an XML Node content value...

Reply:
Yes, right, my problem was actually how to include it as an XML Node attribute

Reply:
> But I can imagine we add `asAttribute` attribute which will place the value into a given attribute of the node.

Ah ok, if that's possible then all right!

Reply:
This is now possible with:
```xml
<dha:RepoObject val="URI" asAttribute="rdf:about">...</dha:RepoObject>
```